src/Files.App/Views/Shells/ColumnShellPage.xaml.cs

```diff
@@ -139,6 +139,11 @@ private async void KeyboardAccelerator_Invoked(KeyboardAccelerator sender, Keybo
 					if (!ToolbarViewModel.IsEditModeEnabled && !ContentPage.IsRenamingItem && !InstanceViewModel.IsPageTypeSearchResults && !ToolbarViewModel.SearchHasFocus)
 						await UIFilesystemHelpers.PasteItemAsync(FilesystemViewModel.WorkingDirectory, this);
 					break;
+
+				// Ctrl + Shift + P, Toggle Command Palette
+				case (true, true, false, true, VirtualKey.P):
+					ToolbarViewModel.IsCommandPaletteOpen = !ToolbarViewModel.IsCommandPaletteOpen;
+					break;
```

Comment:
Is this code needed? Ctrl+Shift+P is a command to open the command palette, not a command to toggle.